src/WebJobs.Script.WebHost/Management/LinuxSpecialization/RunFromPackageHandler.cs

```diff
@@ -134,6 +159,11 @@ private CodePackageType GetPackageType(string filePath, RunFromPackageContext pk
                 return CodePackageType.Squashfs;
             }
 
+            if (pkgContext.IsRunFromLocalPackage())
+            {
+                return CodePackageType.Zip;
```

Comment:
>.Zip [](http://example.com/codeflow?start=38&length=4)

since the package is being generated on the server, it should use squashfs (it should already be the case, please check)

Reply:
I may not be understanding it correctly. On the deployment side, it is saving the zip file provided by client. The server doesn't generate the file.

Reply:
if a remote build is done then it wont be the file provided by client instead it will be a new file generated on the server

[https://github.com/Azure-App-Service/KuduLite/blob/d4bb543f70026fa87a6975c5eb9ff9c5fa350355/Kudu.Core/Helpers/LinuxConsumptionDeploymentHelper.cs#L375](https://github.com/Azure-App-Service/KuduLite/blob/d4bb543f70026fa87a6975c5eb9ff9c5fa350355/Kudu.Core/Helpers/LinuxConsumptionDeploymentHelper.cs#L375)

Reply:
Switched to squashfs  for Linux Consumption.